Tests/MongoSwiftTests/ChangeStreamTests.swift

```diff
@@ -0,0 +1,139 @@
+import mongoc
+@testable import MongoSwift
+import Nimble
+import XCTest
+
+final class ChangeStreamTest: MongoSwiftTestCase {
```

Comment:
these tests are a good starting point. I would say it might be worth adding some assertions on the `fullDocument` field (for updates, inserts, and some ops that are neither). You can wait to adding those tests until later PRs if you would like.

Reply:
I added some assertions on `fullDocument` for update and insert ops. Will add more in the next PR.